src/components/__tests__/login-form-test.jsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-env jest */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import LoginPanel from '../login-form/login-form';
+import WebMonitoringDb from '../../services/web-monitoring-db';
+
+describe('login-form', () => {
+  const getMockedApi = (overrides={}) => Object.assign(
+    Object.create(WebMonitoringDb.prototype),
+    overrides
+  );
+
+  it('Renders a label and input for email', () => {
+    const panel = shallow(<LoginPanel />);
+    const label = panel.find('label').at(0);
+
+    expect(label.text()).toMatch(/e-?mail/i);
+    expect(label.find('input').props().type).toBe('text');
```

Comment:
Your point about autofill was a good one! Feel free to add it back in here (and in the next test) if you want.